.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI
```

Comment:
it should be possible to combine this with the master (previously ci) workflow adding a condition to the publish job to restrict it to pushes to the master branch.

See https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepsif, but for a job instead of a step (it exists on AZP)

Reply:
although undocumented for job,`if` works! will make the change.